networking.md

```diff
@@ -4,17 +4,17 @@ title: Making HTTP Networking Requests in Flutter
 permalink: /networking/
 ---
 
-This page discusses how to make HTTP networking requests in Flutter. For
+This page describes how to make HTTP networking requests in Flutter. For
 HTTP servers and sockets, see [dart:io][dartio].
```

Comment:
Was doing a quick pass, w/o time to research exactly what we were saying here. This gets my vote.